### PR TITLE
Rplace the expected value of a floating point calculation with an exact  value.

### PR DESCRIPTION
**Pull request by luyahan**
Computer calculates  floating-point numbers is always inaccurate, so should manually calculate the expected value.

### CODE REVIEW

**Comment by luyahan** [comment]
#30

File: test/cctest/test-assembler-riscv64.cc [hunk 1]
```diff
@@ -705,13 +727,13 @@ UTEST_R1_FORM_WITH_RES_F(fsqrt_s, float, 34.13f, sqrtf(34.13f))
 UTEST_R2_FORM_WITH_RES_F(fmin_s, float, -1012.0f, 3456.13f, -1012.0f)
 UTEST_R2_FORM_WITH_RES_F(fmax_s, float, -1012.0f, 3456.13f, 3456.13f)
 UTEST_R3_FORM_WITH_RES_F(fmadd_s, float, 67.56f, -1012.01f, 3456.13f,
-                         (67.56f * (-1012.01f) + 3456.13f))
+                         -64915.2656)  // 67.56f * -1012.01f + 3456.13f
```
Comment (fw-brice) [inline test/cctest/test-assembler-riscv64.cc:730]: Is this change still necessary with the change to `ValidateResult`?

Comment (penguinwu) [reply test/cctest/test-assembler-riscv64.cc:730]: if the fix is because of the precision difference between `fma` and `+ *`, then use `std::fma` to compute the expected result would be easier to understand.

Comment (luyahan) [reply test/cctest/test-assembler-riscv64.cc]: Thank you, you are right. I had fix it.

Comment (fw-brice) [reply test/cctest/test-assembler-riscv64.cc]: I thought that the change to `ValidateResult` should be kept, to allow for acceptable variance in the implementation.

Comment (luyahan) [reply test/cctest/test-assembler-riscv64.cc]: I had fix it. Only replacing *+ with std::fma
